verification/curator-service/api/openapi/openapi.yaml

```diff
@@ -167,6 +167,32 @@ paths:
           $ref: "#/components/responses/404"
         "500":
           $ref: "#/components/responses/500"
+  /sources/{id}/retrieve:
+    parameters:
+      - name: id
+        in: path
+        description: The source ID for which to retrieve content
+        required: true
+        schema:
+          type: string
+          pattern: '^[a-f\d]{24}$'
+    post:
+      tags: [Source]
+      summary: Trigger a retrieval of the content for the specified source
+      operationId: retrieve
+      responses:
+        "200":
+          $ref: "#/components/responses/200Retrieval"
+        "400":
+          $ref: "#/components/responses/400"
+        "403":
+          $ref: "#/components/responses/403"
+        "404":
+          $ref: "#/components/responses/404"
+        "422":
+          $ref: "#/components/responses/422"
```

Comment:
This doesn't throw a 422, I don't think.

Reply:
correct, removed.